### PR TITLE
BTFS-1771 add a testnet node; BTFS-1862 IP change

### PR DESCRIPTION
**Pull request by cyberwalker82**
Added a testnet node, and changed IP of a bootstrap node due to networking issue

### CODE REVIEW

**Comment by richlee33** [comment]
we want to also migrate existing config files that contain the old bootstrap testnet and prod nodes.  please see these lines:
https://github.com/TRON-US/go-btfs-config/blob/71e48d3969a149b3b13d885cbb9144d929fca7ee/migrations.go#L59
https://github.com/TRON-US/go-btfs-config/blob/71e48d3969a149b3b13d885cbb9144d929fca7ee/migrations.go#L78

**Comment by cyberwalker82** [comment]
Added the removed IP to known obsolete

File: migrations.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ func migrate_4_SwarmKey(cfg *Config) bool {
 func migrate_5_Bootstrap_node(cfg *Config) bool {
 	obns := []string{
 		"3.120.224.94",
+		"18.196.49.234",
```
Comment (taiyangc) [inline migrations.go:60]: @cyberwalker82 Need to modify `migrate_7_Testnet_Bootstrap_node` as well for testnet node ip update

Comment (taiyangc) [reply migrations.go:60]: You can try `13.59.69.165/tcp/43113` since the ip is not unique

Comment (cyberwalker82) [reply migrations.go:60]: Made the changes to testnet bootstrap nodes

**Pull request merged**